docs/reference/changelog.rst

```diff
@@ -70,6 +70,102 @@ Changelog
 
   For a complete list of commits, check out the `X.Y.Z`_ release on GitHub.
 
+8.5.0 (2024-Nov-19)
+-------------------
+
+Core
+====
+
+* Remove the Snapcraft ``Dockerfile`` in favour of the `snapcraft-rocks`_
```

Comment:
```suggestion
* Remove the Snapcraft Dockerfile in favour of the `snapcraft-rocks`_
```

It's a subtle distinction, but "Dockerfile" is a generic noun rather than a file name, like how we say "README".